CHANGES.rst

```diff
@@ -12,6 +12,10 @@ Cubeviz
 Imviz
 ^^^^^
 
+- Aperture photometry (previously "Imviz Simple Aperture Photometry") now supports batch mode. [#2465]
```

Comment:
The methods on this plugin were not previously public, but I guess technically the default ones like `show` and `open_in_tray` were and would be affected, so that's probably a good idea.